fast64_internal/f3d/f3d_writer.py

```diff
@@ -129,7 +129,7 @@ def fixLargeUVs(obj):
 
 	texSizeDict = {}
 	if len(obj.data.materials) == 0:
```

Comment:
should this also be checking if all applied materials are F3D? cus maybe the error message should instead say "object has zero materials" to be more accurate